### PR TITLE
Update clonetb hash

### PR DESCRIPTION
**Pull request by silabs-krdosvik**
image

### CODE REVIEW

**Review by silabs-robin — COMMENTED** [review]
Does ci_check reasonably pass and is formal still able to run?

File: bin/clonetb
Comment (silabs-robin) [inline bin/clonetb:1]: Does ci_check reasonably pass and is formal still able to run?

I guess it would, because the latest changes to x-dv were minor.

Comment (silabs-krdosvik) [reply bin/clonetb:1]: Formal compiles. Sim dont compile on cv32e40x/mater. Need to wait for this PR to be merged: https://github.com/openhwgroup/cv32e40x/pull/890

Comment (silabs-robin) [reply bin/clonetb:1]: Ok. Let's wait then. And when it is in, it seems we must update the core hash (ExternalRepos) at the same time.

**Pull request closed without merging**